src/compass/s1_geocode_slc.py

```diff
@@ -89,6 +92,19 @@ def run(cfg: GeoRunConfig):
             burst.slc_to_vrt_file(temp_slc_path)
             rdr_burst_raster = isce3.io.Raster(temp_slc_path)
 
+            # Check if EAP correction is necessary; Apply the correction if yes
```

Comment:
May be "if it is True"?

Reply:
Yes, I think your suggestion looks more clear. The comment was revised.